python/test/unit/language/test_core.py

```diff
@@ -351,7 +350,7 @@ def kernel(X, SIZE: tl.constexpr):
 
     check_type_supported(dtype_x, device)
     x = to_triton(numpy_random(SIZE, dtype_str=dtype_x), device=device, dst_type=dtype_x)
-    kernel[(1, )](x, SIZE=SIZE, num_warps=4)
+    kernel.warmup(x, SIZE=SIZE, num_warps=4, grid=(1, ))
```

Comment:
I think this is meant to test that we can successfully compile and run a kernel.